tutorial.md

```diff
@@ -44,6 +44,8 @@ Note: Please create a key with Empty Passphase. We are working on adding support
 ```shell
 gpg --quick-generate-key --yes my.attestator@example.com
 
+(if you use Ubuntu on GCP and got hung in above, then please see https://delightlylinux.wordpress.com/2015/07/01/is-gpg-hanging-when-generating-a-key/ to make sure you got enough entropy.)
```

Comment:
usually, people just copy all the commands in the shell block and  paste them. 
They will blindly copy your amazing Tip and it will error out on the terminal.
Can you please move the tip after the shell block ends @L52 or
 1. end the shell block after `gpg --quick-generate-key` at L47. Add you tip and the begin the shell for the following command at L48